step.yml

```diff
@@ -34,3 +34,17 @@ inputs:
         * If Fastfile path: `./here/is/my/fastlane/Fastfile`
         * Then Fastfile's directory: `./here/is/my/fastlane`
         * So Working Directory should be: `./here/is/my`
+  - update_fastlane: "true"
+    opts:
+      title: "Should update fastlane gem before run?"
+      description: |-
+        Should update fastlane gem before run?
```

Comment:
*This option will be skipped in you have a `Gemfile` in the `work_dir` directory.*